app/lib/settings/src/about_page/about_page_contributors.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:schulplaner8/Helper/GitHub.dart';
+import 'package:schulplaner8/Helper/helper_views.dart';
+import 'package:schulplaner_translations/schulplaner_translations.dart';
+import 'package:schulplaner_widgets/schulplaner_forms.dart';
+
+class AboutContributors extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: MyAppHeader(
+        title: getString(context).about_contributors,
+      ),
+      body: SingleChildScrollView(
+        child: Column(
+          children: [_AboutContributors(), _Contributors()],
+        ),
+      ),
+    );
+  }
+}
+
+class _AboutContributors extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return FormSection(
+        title: BothLangString(de: 'Mitwirkende am Schulplaner', en: 'Schoolplanner\'s collaborators').getText(context),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            FormSectionText(
+                text: DefaultTextSpan(
+              context,
+              BothLangString(
+                      de: 'Der Schulplaner ist Open-Source, sodass jeder daran mitwirken kann. Unten findet ihr eine Liste, welche dynamisch direkt von GitHub geladen wurde und alle Mitwirkenden beinhaltet! Klicke sie an, um zu ihrem GitHub Account zu gelangen!',
+                      en: 'The school planner is open source, so anyone can contribute to it. Below you can find a list, which was dynamically loaded directly from GitHub and includes all contributors! Click them to go to their GitHub account!')
+                  .getText(context),
+            )),
+            SizedBox(
+              height: 20.0,
+            )
+          ],
+        ));
+  }
+}
+
+class _Contributors extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 70, //todo multiply with length
+      child: _myListView(context),
+    );
+  }
+}
+
+Widget _myListView(BuildContext context) {
```

Comment:
This should be a stateless widget. I will add an Comment how to implement this.

Reply:
```
class _ContributorsList extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return FutureBuilder<List<Contributor>>(
      future: getContributors(),
      builder: (context, snapshot) {
        if (!snapshot.hasData) return CircularProgressIndicator();

        final contributors = snapshot.data;
        //Because the value ca
        if (contributors == null) return CircularProgressIndicator();

        return ListView.builder(
          itemCount: contributors.length,
          itemBuilder: (context, index) {
            final _contributor = contributors[index];
            return _ContributorListTile(contributor: _contributor);
          },
        );
      },
    );
  }
}
```